nn_meter/predictor/prediction/extract_feature.py

```diff
@@ -59,7 +59,6 @@ def get_predict_features(config):
             #features = [inputh, len(itensors)]
             features = [inputh]
             for it in itensors:
-                #co = it[-1]
```

Comment:
This part where we are extracting the channels from a tensor, it depends on who the features are actually composed in a layer, few models have { n , c , h , w } and few models have { n , h , w , c } hence 
`co = it[-1]` or `co = it[-2]` or `co = it[1]` matters on the model we are sending for the prediction so can you please look into this very soon !!